docs/tutorial/using-docker-container.md

```diff
@@ -62,12 +58,12 @@ the default version of WSL is 1 after installing, it needs to upgrade to version
 1. please upgrade to version 2, if not
 
    ```c
-   wsl --set-version Ubuntu-20.04 2
+   wsl --set-version Ubuntu 2
    ```
 
-1. set the distribution, as below:
+1. set the distribution as default, as below:
```

Comment:
I think it would sound better if we avoid the double "as". Maybe something like:
"configure the distribution to be the default option, as specified below:"